pkgs/development/python-modules/ddddocr/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  numpy,
+  onnxruntime,
+  pillow,
+  opencv-python-headless,
+}:
+
+buildPythonPackage rec {
+  pname = "ddddocr";
+  version = "1.5.6";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-KDmpQL+r4C4yhO8/nSoDcpKqn2QfNVtDqbcL7Onhtz0=";
+  };
```

Comment:
fetchFromGitHub is preferred

Reply:
They don't version their github (no releases, no tags, no branches), it would be manually moving commit hashes every release (and I expect the bot wouldn't be able to help).  By using pypi we can move between actual meaningful version numbers.